source/features/disable-custom-colors.js

```diff
@@ -0,0 +1,6 @@
+export default () => {
+	if (document.body.classList.contains('ProfilePage')) {
+		const userStyles = $('head')[0].querySelector('style');
```

Comment:
`const userStyles = document.head.querySelector('style');`

Reply:
And I think just getting the first `<style>` element is too fragile. This could easily change. Better to target the ID `user-style-sindresorhus`.